tests/queries/0_stateless/00804_test_delta_codec_compression.sql

```diff
@@ -115,3 +115,41 @@ USING(key);
 
 DROP TABLE IF EXISTS delta_codec_string;
 DROP TABLE IF EXISTS default_codec_string;
+
```

Comment:
00804_test_delta_codec_compression basically tests delta compression. I don't think we need to change it. 

(besides that I don't understand why a test named "_string" (123, 128) tests floats (l. 125, 130).)

Reply:
Done